index.js

```diff
@@ -0,0 +1,138 @@
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+
+const loadTrips = () => {
+
+  const tripList = $('#trip-list');
+
+  axios.get(URL)
+    .then((response) => {
+      response.data.forEach((trip) => {
+        let tripID = trip.id;
+          tripList.append(`<li id="${tripID}">${trip.name}</li>`);
+      });
+    })
+    .catch((error) => {
+      console.log(error);
+    });
+};
+
+
+const loadDetails = (id) => {
+  let idURL = `${URL}/${id}`;
+
+  axios.get(idURL)
+
+    .then((response) => {
+
+      if(response.data) {
+        // log info to console
```

Comment:
It might make sense to break the code to render trip details out into a separate function - as is, this function is doing two things (making the request, and rendering data). See also http://callbackhell.com/